config/env/default.js

```diff
@@ -18,6 +18,16 @@ module.exports = {
     description: 'Travellers community for sharing, hosting and getting people together. We want a world that encourages trust and adventure.'
   },
 
+  // Feature flags
+  featureFlags: {
+    // enable references at all?
+    reference: false
```

Comment:
This flag is probably enough for now, detailed ones below we can start implementing/using once we publish the first version of this.